src/intro_gui.cpp

```diff
@@ -184,7 +184,7 @@ struct SelectGameWindow : public Window {
 
 		this->ReadIntroGameViewportCommands();
 
-		this->cur_viewport_command_index = (size_t)-1;
+		this->cur_viewport_command_index = static_cast<size_t>(-1);
```

Comment:
```suggestion
		this->cur_viewport_command_index = std::numeric_limits<size_t>::max();
```

also a couple of other places

Reply:
Or just `SIZE_MAX`?

Reply:
maybe.